docs/mindsdb-docs/docs/sql/project.md

```diff
@@ -0,0 +1,368 @@
+# The `PROJECT` Entity
+
+MindsDB introduces the `PROJECT` entity that lets you create projects to store your models.
+
+## Working with `PROJECTS`
+
+### Creating `PROJECTS`
+
+You can create projects to store your models, making the structure like:
+
+```
```

Comment:
Missing the models table and the models_verions table

Reply:
Added.